types/consola.d.ts

```diff
@@ -1,10 +1,35 @@
+import { InspectOptions } from 'util';
+import {Logger} from 'winston';
+
+export type Infinity = 999e308; // Infinity is not a valid type. see https://github.com/microsoft/TypeScript/issues/32277
+export type FatalLogLevel = 0;
+export type ErrorLogLevel = 0;
+export type WarnLogLevel = 1;
+export type LogLogLevel = 2;
+export type InfoLogLevel = 3;
+export type SuccessLogLevel = 3;
+export type DebugLogLevel = 4;
+export type TraceLogLevel = 5;
+export type SilentLogLevel = Infinity;
+export type LogLevel =
```

Comment:
Can we instead use an enum?

Reply:
Enums won't work well on d.ts alone as you must expose a real object to represent them. I will see if I can add it to the js and then expose it as an Enum here

Reply:
Fixed it, can you please CR it again?